buildSrc/src/main/java/Config.kt

```diff
@@ -45,9 +45,11 @@ object Plugins {
     const val androidPlugin = "android"
     const val appPlugin = "com.android.application"
     const val kapt = "kapt"
+    const val multiplatform = "multiplatform"
```

Comment:
I'd suggest we introduce all new libraries/plugins here

Reply:
please separate files for libs/plugins/modules
example:
https://github.com/game-x50/android_client_app/tree/master/buildSrc/src/main/java

apply gradle  version catalog:
https://sourcediving.com/manage-your-gradle-dependencies-with-version-catalog-not-only-in-android-114117647cdb

enable TYPESAFE_PROJECT_ACCESSORS
example:
https://github.com/game-x50/android_client_app/blob/master/settings.gradle.kts#L1#L2

Reply:
Done, I think)